maro/simulator/scenarios/data_center/virtual_machine.py

```diff
@@ -34,11 +35,11 @@ def get_utilization(self, cur_tick: int):
 
         return self._utilization_series[cur_tick - self.start_tick]
 
-    def add_utilization_series(self, _utilization_series: List[float]):
+    def add_utilization(self, cpu_utilization: float):
         """VM CPU utilization list."""
-        self._utilization_series = _utilization_series
+        self._utilization_series.append(cpu_utilization)
+        self._utilization_index = len(self._utilization_series) - 1
```

Comment:
what's the meaning of ``utilization_index``? points to the last element in read-in utilization?
If so, the return value of ``get_historical_utilization_series`` would always be the read-in part, which may include both historical and future series.

If the ``utilization_index`` points to the current_utilization, it would be equal to ``cur_tick - self.start_tick``?

Reply:
Yes, so here are some typos. The utilization index is used for the current utilization, not for the end of the utilization series.